+mystica/+state/@StateKinMBody/StateKinMBody.m

```diff
@@ -56,7 +59,20 @@ function clearProperties(obj)
         end
 
         function intJcV = getIntJcV(obj)
-            intJcV = full(obj.csdFn.intJcV(obj.mBodyPosQuat_0,obj.mBodyPosQuat_0_initial));
+            intJcV = full(mystica_stateKin('intJcV',obj.mBodyPosQuat_0,obj.mBodyPosQuat_0_initial)); % obj.csdFn.intJcV(obj.mBodyPosQuat_0,obj.mBodyPosQuat_0_initial)
+        end
+
+        function generateMEX(obj)
+            opts = struct('main', true,'mex', true);
+            C = casadi.CodeGenerator('mystica_stateKin.c',opts);
+            C.add(obj.csdFn.Jc);
+            C.add(obj.csdFn.intJcV);
+            C.add(obj.csdFn.rC_from_jointsAngVelPJ_2_jointsAngVel0);
+            C.add(obj.csdFn.rC_from_mBodyTwist0_2_jointsAngVelPJ);
+            C.add(obj.csdFn.get_mBodyVelQuat0_from_mBodyTwist0)
+            C.generate();
+            mex mystica_stateKin.c -largeArrayDims
```

Comment:
For curiosity :)
Once the function `mystica_stateKin` is generated, does it live in the workspace and be called in every part of the code?

Reply:
Also, is it computationally expensive to generate it? 
I might understand why you don't check if the function already exists. 
To check if the content of the function changed, you have to generate it again :D

Reply:
I generate the MEX files in the current folder. So, until you do not change the current directory, they can be called everywhere